site/docs/config.md

```diff
@@ -198,7 +198,7 @@ vg.embed('#horizontal_line', {
 
 | Property      | Type          | Description    |
 | :------------ |:-------------:| :------------- |
-| barWidth      | Number        | The width of the bars. <span class="note-line">__Default value:__  `bandWidth-1` for the bar's dimension is an ordinal scale. (This provides 1 pixel offset between bars.) and `2` for if bar's dimension has linear scale.  </span>  |
+| barWidth      | Number        | The width of the bars. <span class="note-line">__Default value:__  `bandSize-1` for the bar's dimension is an ordinal scale. (This provides 1 pixel offset between bars.) and `2` for if bar's dimension has linear scale.  </span>  |
```

Comment:
But bar's size channel is always this width thing.   (Length is fixed)

But I was thinking about this the other day too.   (Should we allow using size for gantt bar?)  The current meaning of size for bar is mostly inspired from Tableau, but has no practical use.